recipes/rangesnext/all/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class RangesnextConan(ConanFile):
+    name = "rangesnext"
+    description = "ranges features for C++23 ported to C++20"
+    topics = ("conan", "rangesnext", "ranges", "backport", "backport-cpp")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cor3ntin/rangesnext"
+    license = "BSL-1.0"
+    exports_sources = "patches/**"
+    settings = "compiler"
+
+    _compilers_minimum_version = {
+        "gcc": "10",
+        "Visual Studio": "19",
+        "clang": "13"
+    }
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

Validate became official since Conan 1.32.0. You still will find recipes using `configure`, but consider it as outdated. There is a blog post about: https://blog.conan.io/2020/12/14/New-conan-release-1-32.html